Mobile Test Runner for Jama/Mobile Test Runner for Jama/ProjectListViewController.swift

```diff
@@ -48,6 +63,39 @@ extension ProjectListViewController: EndpointDelegate {
         }
         DispatchQueue.main.async {
             self.projectList.extractProjectList(fromData: unwrappedData)
+            
+            //It looks like the API returns the list sorted but it seems like we should make sure
+            self.projectList.projectList.sort(by: self.compareProjectNames(lhs:rhs:))
+//            self.debugHugeProjectList()
+            self.collectionView.reloadData() //After async call, reload the collection data
         }
     }
 }
+
+
+extension ProjectListViewController: UICollectionViewDelegate, UICollectionViewDataSource {
+    //How many cells will the collection view have
+    func collectionView(_ collectionView: UICollectionView, numberOfItemsInSection section: Int) -> Int {
+        return projectList.projectList.count
+    }
+    
+    //Makes a cell for the collection. This is called for each of the projects in the list.
+    func collectionView(_ collectionView: UICollectionView, cellForItemAt indexPath: IndexPath) -> UICollectionViewCell {
```

Comment:
@lcc4 added some tests to cover this and the other missing coverage. Extracted some code to a its own method since it is all related.